examples/gcp-pubsub.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: pubsub.cnrm.cloud.google.com/v1beta1
+kind: PubSubTopic
+metadata:
+  annotations:
+    velocity.tech.v1/id: pubsub-topic # Velocity service identifier
+    velocity.tech.v1/exports-topic-name: path="metadata.name"
+    cnrm.cloud.google.com/project-id: "{velocity.v1.gcp.projectName}"
```

Comment:
I think that this is optional,
if you omit this line we automatically add it for you